### PR TITLE
Travelcost

### PR DESCRIPTION
**Pull request by Uschi760**
For different reasons - I added travel costs just on confirmation page - (to display it on form - AJAX is necessary) - and i left the starship costs because of the missing connection between trip and starship in the model ...

### CODE REVIEW

File: app/assets/stylesheets/bodies/_show.scss [hunk 1]
```diff
@@ -7,6 +7,34 @@
...
+  margin-bottom: 20px;
+
+  h3 {
+    text-align: center;
+    padding-bottom: 10px;
+    margin-bottom: 20px;
+  }
+}
+
+.content-details {
```
Comment (gitovska) [inline app/assets/stylesheets/bodies/_show.scss:38]: you have selected the css class -content-details twice. you can merge this

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -18,8 +18,55 @@
     </div>
     <% end %>
   </div>
+
```
Comment (gitovska) [inline app/views/pages/home.html.erb:21]: these changes don't seem to be related to the travel cost. is this part of @d-tello 's PR for the homepage edits?

File: app/views/trips/show.html.erb [hunk 3]
```diff
@@ -5,13 +5,26 @@
...
         <%= @trip.departure_date %>
       </li>
+
+
+<li>
+      <% arrival_start = @trip.arrival_date %>
+      <% departure_start = @trip.departure_date %>
+      <% days_in_range = (arrival_start .. departure_start).count %>
+
+      <% bodie_price = @trip.body.cost_per_day %>
```
Comment (gitovska) [inline app/views/trips/show.html.erb:23]: body_price instead of bodie_price

**Review by gitovska — CHANGES_REQUESTED** [review]
There seem to be files in this branch that don't relate to travel cost. I have also made some other comments. Looking good though!

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (Uschi760) [reply app/views/pages/home.html.erb:21]: I was not changing something here ... so Im not sure about it ...

File: app/views/trips/show.html.erb (hunk 3 above)
Comment (Uschi760) [reply app/views/trips/show.html.erb:23]: oh yeah sorry ... ill gonna change it ..
